casr/src/triage.rs

```diff
@@ -70,32 +71,10 @@ impl<'a> CrashInfo {
             args[at_index + offset] = input;
         }
 
-        let mut casr_cmd = Command::new(tool);
-        casr_cmd.args(&args);
-        casr_cmd.envs(&self.envs);
-
-        // Add envs
-        if self.target_args.iter().any(|x| x.eq("-detect_leaks=0")) {
```

Comment:
Why do we remove this code?

Reply:
We ddn't, just move it to the upper function